neqo-transport/src/connection/mod.rs

```diff
@@ -2817,6 +2811,36 @@ impl Connection {
         }
     }
 
+    fn pad_initial(
+        &self,
+        encoder: &mut Encoder<&mut Vec<u8>>,
+        initial: &mut sent::Packet,
+        profile: &SendProfile,
+    ) {
+        if encoder.len() >= profile.limit() {
+            return;
+        }
+
+        qdebug!(
+            "[{self}] pad Initial from {} to {}",
+            encoder.len(),
+            profile.limit()
+        );
+        let pad_amount = profile.limit() - encoder.len();
+        initial.track_padding(pad_amount);
+        // This ensures that the last bytes are a SCONE indication, if there is enough space.
+        // This is not tracked, other than for congestion control (above)
+        if pad_amount >= Self::SCONE_INDICATION.len() {
+            encoder.pad_to(
+                profile.limit() - Self::SCONE_INDICATION.len() + 1,
+                Self::SCONE_INDICATION[0],
```

Comment:
I understand we want to change the last two padding bytes to `0xc8` and `0x13`. 

Why change all other padding bytes to `0xc8`? For performance?

Reply:
Not performance, but I figured that it would be worth a change.  It changes how we treat the padding bytes (0xc8 is a long header packet), which should help with robustness tests on other implementations.